### ISSUE TITLE
Incorrect town for dividing roadway

### ISSUE THREAD

**Issue by mikemazurkiewicz** [openstreetmap/Nominatim#1281]
I was searching using the following url:

https://nominatim.openstreetmap.org/search.php?q=1146+maple+ave%2C+la+grange+park%2C+il&polygon_geojson=1&viewbox=

The results display the address on the correct side of the roadway, but in the details, the address is shown as Brookfield, not La Grange Park.

The detail at: https://nominatim.openstreetmap.org/details.php?place_id=210950133 shows that it is listed in Brookfield. Maple Ave from 26th St to the north and 31st st to the south is the boundary for La Grange Park (west side of street) and Brookfield (east side of the street).

**Comment by mtmail** [comment]
The position of the house number comes from a supplementary dataset [US TIGER](http://nominatim.org/release-docs/develop/data-sources/US-Tiger/). That additional table is searched, but the records in there have no `place_id`, `osm_id`, `osm_type`. Because of that link to the details page goes to the next higher map feature (the road).

On the details page https://nominatim.openstreetmap.org/details.php?osmtype=W&osmid=24085033&class=highway the road is just on the border of both `place=village` boundaries, without house number one of the villages is chosen.

**Comment by mikemazurkiewicz** [comment]
Hi,
Ok, I searched for:
1146 maple ave, la grange park, il

The results showed:
1146, Maple Avenue, East Grossdale, Brookfield, Cook County, Illinois, 60526, USA (House)

This was from the first link that I documented. The map shows it being on the La Grange Park side of the street, not Brookfield. I think there is a bug since the map shows the town/village boundary, the pin is in the correct location on the map, but the wrong town is documented in the text.

I understand that the second link was for details of the roadway with no reference to the house number. I should not have included it as I was having problems with the initial search link not the second link.
